### PR TITLE
fixes #23

### LINKED ISSUE #23: Can you make it clear that we can't provide housing?
Something along the lines of:
- we want to try our best to make hack camp great for anyone who comes
- right now it is not in our ability to provide housing

@MaxWofford @zachlatta

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by maxwofford** [comment]
@zachlatta @jonleung Are we still including these questions?

File: application.html [hunk 1]
```diff
@@ -182,6 +189,13 @@ <h1 class="white title">Apply to Hack Camp</h1>
                 </li>
               </ul>
             </div>
+            <div class="small-12 columns">
+              <label>
+                Can you briefly (1-3 sentences) tell us about something you've
+                coded before and how it worked?
+                <input name=" entry.159477422" type="text" />
```
Comment (zachlatta) [inline application.html]: Can you remove the space in the name and have this field only shown if they selected "I've completed a complete project" or more experience?

@jonleung thoughts?

**Comment by maxwofford** [comment]
Thanks @zachlatta. Will ping when ready for merge.

**Comment by maxwofford** [comment]
@zachlatta This should be good to go.

File: application.html [hunk 2]
```diff
@@ -135,6 +135,13 @@ <h1 class="white title">Apply to Hack Camp</h1>
             </div>
           </div>
           <div class="row">
+            <div class="columns">
+              <label>In 2-3 sentences, why do you want to go to Hack Camp?
+                <input name="entry" type="text" />
```
Comment (zachlatta) [inline application.html]: Is this the right input field name?

**Comment by zachlatta** [comment]
Closing for the time being, we're all too sleepy to write/review proper code.

**Comment by maxwofford** [comment]
@zachlatta Ping

**Comment by zachlatta** [comment]
See existing comment.

**Comment by maxwofford** [comment]
Oops, my bad. Amended and pushed. @zachlatta

**Pull request merged**